src/meta/meta_bulk_load_service.h

```diff
@@ -40,13 +40,17 @@ struct app_bulk_load_info
     std::string file_provider_type;
     bulk_load_status::type status;
     std::string remote_root_path;
+    bool is_ever_ingesting;
+    error_code bulk_load_err;
```

Comment:
I suggest using `bulk_load_result`

Reply:
This param is useful only if status=bulk_load_status::BLS_FAILED, so it stand for "error", not "result"